javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -109,18 +118,21 @@ private module Postgres {
     result = newPool().getMember("connect").getParameter(0).getParameter(1)
   }
 
-  /** Gets an expression that constructs a new connection pool. */
-  API::Node newPool() {
+  /** Gets a constructor that when invoked constructs a new connection pool. */
+  API::Node newPoolCallee() {
     // new require('pg').Pool()
-    result = API::moduleImport("pg").getMember("Pool").getInstance()
+    result = API::moduleImport("pg").getMember("Pool")
     or
     // new require('pg-pool')
-    result = API::moduleImport("pg-pool").getInstance()
+    result = API::moduleImport("pg-pool")
   }
 
+  /** Gets an expression that constructs a new connection pool. */
+  API::Node newPool() { result = newPoolCallee().getInstance() }
```

Comment:
Suggest renaming to `pool`, with `newPoolCallee` becoming `newPool` or `poolConstructor`.